cli/index.js

```diff
@@ -86,16 +86,19 @@ if (opts['server-option']) {
   opts.server_options = opts['server-option'];
 }
 
-(async () => {
-  if (opts._.length) {
+if (opts._.length) {
+  if (opts._[0] === '-') {
+    opts.spec = process.stdin;
+  } else {
     opts.spec = opts._;
   }
-  delete opts._;
-  try {
-    const { exit_code } = await mochify(opts);
+}
+delete opts._;
+mochify(opts)
```

Comment:
Unwrapping this and using a Promise chain is here to stop eslint from complaining. I'm not entirely sure if its complaints are correct or not yet. We could also keep the old version and ignore the rule, no real preference from my side here.

See https://github.com/eslint/eslint/issues/15076

Reply:
Probably even easier to read than the self-invoking async function. 👍